Formula/ledger.rb

```diff
@@ -28,6 +28,7 @@ class Ledger < Formula
   depends_on "gmp"
   depends_on "mpfr"
   depends_on "python@3.10"
+  depends_on "texinfo"
```

Comment:
Can it use the one from macOS?

Reply:
I don't believe it can as it was causing an error for me:

```
==> ./acprep opt make doc --jobs=8 --output=build --prefix=/usr/local/Cellar/ledger/HEAD-0eac2c4_10 --boost=/usr/local/opt/boost -- -DBUILD_DOCS=1 -DBUILD_WEB_DOCS=1 -DBoost_N
Last 15 lines from /Users/kfoley/Library/Logs/Homebrew/ledger/02.acprep:
/tmp/ledger-20221228-36344-yxa39n/doc/ledger3.texi:9: Unknown command `codequoteundirected'.
/tmp/ledger-20221228-36344-yxa39n/doc/ledger3.texi:9: Unknown command `codequoteundirected'.
/tmp/ledger-20221228-36344-yxa39n/doc/ledger3.texi:160: warning: unrecognized encoding name `UTF-8'.
/tmp/ledger-20221228-36344-yxa39n/doc/ledger3.texi:160: warning: unrecognized encoding name `UTF-8'.
make[3]: *** [doc/ledger3.info] Error 1
make[3]: *** Deleting file `doc/ledger3.info'
make[3]: *** Waiting for unfinished jobs....
make[3]: *** [doc/ledger3.html] Error 1
make[3]: *** Deleting file `doc/ledger3.html'
mdoc warning: Empty input line #1458
mdoc warning: Empty input line #1458
make[2]: *** [doc/CMakeFiles/doc.dir/all] Error 2
make[1]: *** [doc/CMakeFiles/doc.dir/rule] Error 2
make: *** [doc] Error 2
acprep: ERROR: Execution failed: make doc -Wno-dev -j8
```

Which from my research is caused by using an older version of `texinfo`.

Reply:
What OS version are you running?

Homebrew CI didn't hit any build issue on Catalina, Big Sur, and Monterey since we have bottles for all of those. So, if system `texinfo` was picked up, then the version available was sufficient.

As of Ventura, `texinfo` is no longer available and there is no build issue without it. Probably just skipped.

---

Anyway, the changes aren't correct based on CI results and it is unlikely to be a runtime dependency.

Perhaps some older macOS `texinfo` version can cause build issues if detected by CMake. In that case, dependency should be restricted to older macOS only.

Reply:
> What OS version are you running?

I'm running Big Sur.

However, I didn't mention that I'm building from head.  The latest release of `ledger` is from May 2020. 

I just tested locally and using `--build-from-source` without `--HEAD` runs successfully so there must have been some change since the last release which requires a newer version of `texinfo`.

Reply:
Yeah, that's pretty essential information.